packages/venia-ui/lib/components/HomePage/homePage.module.css

```diff
@@ -419,3 +419,15 @@
 :global(.venia-home-products .slick-slider .slick-dots .slick-active > button) {
     background-color: hsl(205 004% 048%) !important;
 }
+
+:global(.venia-home-banner-text > div) {
+    flex-direction: unset !important;
+    justify-content: normal !important;
+}
+
+@media (max-width: 960px) {
+    :global(.venia-home-banner-text > div) {
+        flex-direction: column !important;
+        justify-content: center !important;
+    }
+}
```

Comment:
I'm don't this is the best approach to css. By using important, it makes it harder to modify css for this in the future.

Reply:
I agree. I always avoid `!important`, however the original CSS on the dynamic blocks is directly added in-line with JavaScript.

The file already had many `!important` for the same reason : https://github.com/magento/pwa-studio/blob/6c8e6f638048be059846bc4b04224845089c50e7/packages/venia-ui/lib/components/HomePage/homePage.module.css